doc/cookbook/source/references.bib

```diff
@@ -163,3 +163,10 @@ @article{hyvarinen2000independent
   pages={411--430},
   year={2000}
 }
+@inproceedings{basak2008kernel
+	author={J. Basak},
+	booktitle={2008 19th International Conference on Pattern Recognition},
```

Comment:
is this really the best reference? where do you have it from? can't we use some of the references we already used (like the kernel book by schölkopf?)

Reply:
i found several links point to this paper, e.g https://github.com/gmum/pykernels/blob/c8afdc79a15197ad3be2a0db0118f5e948577f49/pykernels/regular.py#L132
and I've checked that paper, the author gave a definition of it in sec.3

Reply:
ok cool. When citing it, put [Section 3] in there (if possible)